charts/s1-agent/values.yaml

```diff
@@ -94,6 +97,29 @@ agentInjection:
       cpu: 100m
       memory: 800Mi
 
+admissionControllers:
+  validating:
+    selector:
+      namespaceSelector:
+        matchExpressions:
+          - key: kubernetes.io/metadata.name
+            operator: NotIn
+            values:
+              - kube-system
+              - kube-public
+    rules:
+    - apiGroups:
+      - ""
```

Comment:
I think we might need this to be '*' since it's not empty for all the assets we are interested in (for example: in "StatefulSet" it is "apps").
Alternatively, we can list all the groups of the assets we intend to validate.